service/src/test/java/bio/terra/pipelines/service/JobsServiceTest.java

```diff
@@ -1,83 +1,108 @@
 package bio.terra.pipelines.service;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.ArgumentMatchers.argThat;
-import static org.mockito.Mockito.*;
 
-import bio.terra.pipelines.db.JobsDao;
-import bio.terra.pipelines.service.model.Job;
-import bio.terra.pipelines.testutils.BaseUnitTest;
+import bio.terra.pipelines.db.entities.DbJob;
+import bio.terra.pipelines.db.repositories.JobsRepository;
+import java.time.Instant;
+import java.util.List;
 import java.util.UUID;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.ActiveProfiles;
 
-class JobsServiceTest extends BaseUnitTest {
+@DataJpaTest(properties = "spring.main.lazy-initialization=true")
+@ActiveProfiles({"test", "human-readable-logging"})
+class JobsServiceTest {
 
-  @Autowired private JobsService jobsService;
-  @MockBean private JobsDao jobsDao;
+  @Autowired JobsService jobsService;
+  @Autowired JobsRepository jobsRepository;
+
+  @Autowired TestEntityManager testEntityManager;
 
-  // parameters used repeatedly by various tests, and things we'll want to mocks to respond to
-  // universally
   private final String testUserId = "testUser";
-  private final String testGoodPipelineId = "testGoodPipeline";
-  private final String testPipelineVersion = "testPipelineVersion";
-
-  // We'll need these to configure the dao to return selectively good or bad values
-  private final UUID testGoodUUID = UUID.randomUUID();
-  private final UUID testDuplicateUUID = UUID.randomUUID();
-
-  @BeforeEach
-  void initMocks() {
```

Comment:
most of the removals in this file (all of the mocked out test related stuff) were moved into a new test file called `JobsServiceMockTest` and i put then non mocked tests in here (modified to work with the jpa repository).  No test was removed in this pr, just modified or net new tests